_includes/partners/contact.html

```diff
@@ -406,9 +406,12 @@
 
     <input type="submit" class="usa-button usa-button--big usa-button--wide" value="Submit" />
     <button type="reset" class="usa-button usa-button--big usa-button--wide usa-button--outline">Clear form</button>
-    <div id="form-error-message" class="usa-error-message display-none margin-top-4" role="alert">
+    <div id="form-submit-error-message" class="usa-error-message display-none margin-top-4" role="alert">
       We were unable to submit your request at this time. Please visit our <a href="https://zendesk.login.gov/hc" target="_blank">Partner Support Portal</a> and submit a request for a "New Login.gov Partnership".
     </div>
+    <div id="form-email-error-message" class="usa-error-message display-none margin-top-4" role="alert">
+      We were unable to submit your request at this time. Please make sure to enter your government email address and try again. If you are looking for support with your Login.gov account, please <a href="https://login.gov/contact">contact our user support team</a> for assistance.
```

Comment:
Can we use the site relative URL so that this link works in local development and in Federalist previews?

```suggestion
      We were unable to submit your request at this time. Please make sure to enter your government email address and try again. If you are looking for support with your Login.gov account, please <a href="{{ site.baseurl }}/contact/">contact our user support team</a> for assistance.
```

Reply:
yup, good catch!